apps/server/src/modules/deletion/client/deletion.client.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { of } from 'rxjs';
+import { AxiosResponse } from 'axios';
+import { HttpService } from '@nestjs/axios';
+import { ConfigService } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { axiosResponseFactory } from '@shared/testing';
+import { DeletionRequestInputBuilder, DeletionRequestOutputBuilder } from '@modules/deletion';
```

Comment:
```suggestion
import { DeletionRequestInputBuilder, DeletionRequestOutputBuilder } from './';
```

Reply:
Please use imports within the module, always with a relative path

Reply:
Sure, thanks, I always try to do that, but as the IDE is usually adding these imports for me, it's easy to skip a few imports for some larger portion of code (also I'm not sure why IDEs usually doesn't start with the relative, "closest" imports, but with the non-relative ones...). But you're right, these should always be a relative imports. BTW just `'.'` is enough here 😊 .